proj.sh

```diff
@@ -49,6 +49,7 @@ function deploy-stack {
   CHOSEN_ENV="env-${1:-$LOCALENV}.env"
   generate-data
   docker compose --env-file $CHOSEN_ENV -f docker-compose.yml -f docker-compose-remote.yml --project-name metabolicatlas up --detach --build --force-recreate --remove-orphans --renew-anon-volumes
+  docker compose --env-file $CHOSEN_ENV -f docker-compose.yml -f docker-compose-remote.yml exec -it pg psql -U postgres -c "alter user postgres with password '${POSTGRES_PASSWORD}';"
```

Comment:
This tends to print the output `ALTER ROLE`, which I don't find that informative, so I'm suggesting to hide it with a change in options:
```suggestion
  docker compose --env-file $CHOSEN_ENV -f docker-compose.yml -f docker-compose-remote.yml exec -d pg psql -U postgres -c "alter user postgres with password '${POSTGRES_PASSWORD}';"
```